raiden/ui/runners.py

```diff
@@ -72,6 +72,8 @@ def run(self):
             disable_debug_logfile=self._options['disable_debug_logfile'],
         )
 
+        log.info('Starting Raiden', **get_system_spec())
```

Comment:
`get_system_spec()` can also return `None`. Also question: will this be included in the beginning of the debug log?

Reply:
No it can’t. It may return a None version but it will always return a dictionary.

This log call is directly after we configure logging, so it should be the first line in the log.

Reply:
So the info of the dictionary will be enough to see the commit we are running I guess?

Reply:
Yes, unless the version returned is `None` but that can't happen with binary builds and also shouldn't happen with source installs unless someone has a really broken setup.